test/browser/spec.js

```diff
@@ -124,4 +124,49 @@ describe('Component spec', () => {
 			expect(WithDefaultProps.prototype.getDefaultProps).to.be.calledOnce;
 		});
 	});
+
+	describe('forceUpdate', () => {
+		it('should force a rerender', () => {
+			let forceUpdate;
+			class ForceUpdateComponent extends Component {
+				componentWillUpdate() {}
+				componentDidMount() {
+					forceUpdate = () => this.forceUpdate();
+				}
+				render() {
+					return <div />;
+				}
+			}
+			sinon.spy(ForceUpdateComponent.prototype, 'componentWillUpdate');
+			sinon.spy(ForceUpdateComponent.prototype, 'forceUpdate');
+			render(<ForceUpdateComponent />, scratch);
+			expect(ForceUpdateComponent.prototype.componentWillUpdate).not.to.have.been.called;
+
+			forceUpdate();
+
+			expect(ForceUpdateComponent.prototype.componentWillUpdate).to.have.been.called;
+			expect(ForceUpdateComponent.prototype.forceUpdate).to.have.been.called;
```

Comment:
Thanks for adding these tests, although it doesn't actually check that the component got re-rendered. Ideally the test would do something mirroring a simplified real-world use of `forceUpdate`. Something like:

1. Create a component whose output depends on something other than props/state
2. Render
3. Change the external state
4. Force update
5. Check that the rendered output reflects the updated external state.